app/src/main/java/com/smelnykov/homework_1/MyProfileActivity/MyProfileActivity.kt

```diff
@@ -32,4 +44,34 @@ class MyProfileActivity : AppCompatActivity() {
         val name = binding.appCompatTextName
         name.text = parser.parseNameSurname(email)
     }
+
+    /**
+     * Method sets up the click listeners for log out button.
+     */
+    private fun setClickListeners() {
+
+        val buttonLogOut = binding.appCompatButtonLogOut
```

Comment:
I don't see why you need a variable here

Reply:
Thank you! Made changes. Commit 9e0bcee7